docker-compose.yml

```diff
@@ -325,6 +325,7 @@ services:
       OPENELIS_DB_USER: ${OPENELIS_DB_USER}
       OPENELIS_DB_PASSWORD: ${OPENELIS_DB_PASSWORD}
       OPENELIS_DB_NAME: ${OPENELIS_DB_NAME}
+    command: postgres -c wal_level=logical -c max_wal_senders=10 -c max_replication_slots=10 -c max_connections=200
```

Comment:
> we still need to upgrade the Bahmni Docker Compose project on each instance, right?

Correct.

Reply:
I'm surprised.
Are you sure about this? Isn't it only needed on the server that will do the actual flattening?

Reply:
Yes specifically we need the Kafka to connect to connect to the PostgreSQL analytics database. Without that I was not able to do the Batch job.

Reply:
Sorry, I guess I'm missing something... Do you mean the Odoo database is replicated on a different server from which streams everything required by the analytics suite?

Reply:
If so then, I guess this change is not valid here. I'm not sure where this changes would be made...

Reply:
So we don't have to update all production servers because in the context of C2C, the flattening will happen only on 1 computer.
The data will be restored from the production systems (which run PSQL 6.x) onto a PSQL 13 instance and the batch should work.

Reply:
That means that Stream Mode should work too. I that case the change here is totally unnecessary.